spec/features/post_spec.rb

```diff
@@ -0,0 +1,149 @@
+require 'rails_helper'
+
+RSpec.describe 'Post pages test', type: :feature do
+  describe 'index page' do
+    before :each do
+      @user_first = User.create(name: 'Grabrielle', photo: 'mybaby.png', bio: 'A cute baby', post_counter: 0)
+
+      @post_first = @user_first.posts.new(title: 'Love must lead', text: 'This is my first post')
+      @post_first.likes_counter = 0
+      @post_first.comments_counter = 0
+      @post_first.update_users_posts_counter
+      @post_first.save
+
+      @comment = @user_first.comments.new(post_id: @post_first.id, user_id: @user_first.id, text: 'My first comment')
+      @comment.post_id = @post_first.id
+      @comment.update_comments_posts_counter
+      @comment.save
+
+      @like = Like.new(user_id: @user_first.id, post_id: @post_first.id)
+      @like.user = @user_first
+      @like.post = @post_first
+      @like.save
+      @like.update_posts_likes_counter
+    end
+
+    it 'check posts#index path' do
```

Comment:
- [ ] some of your tests are failing please try to fix them in order to pass without errors
<img width="1140" alt="Screen Shot 2022-08-23 at 10 33 38" src="https://user-images.githubusercontent.com/61790587/186126324-fb7a841e-95e5-47f1-a118-08512908c994.png">